src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -67,13 +78,42 @@ export const InlineEditExample = {
       demo: <InlineEditTitle />,
       props: { EuiInlineEditTitle },
     },
+    {
+      title: 'Customizing read and edit modes',
+      text: (
+        <>
+          <p>
+            Customize the <EuiCode>readMode</EuiCode> empty button by passing{' '}
+            <strong>EuiInlineEdit</strong> the <EuiCode>readModeProps</EuiCode>{' '}
+            property. <EuiCode>readMode</EuiCode> accepts{' '}
```

Comment:
Minor copy tweaks for succinctness
```suggestion
            Customize the <EuiCode>readMode</EuiCode> empty button by passing <EuiCode>readModeProps</EuiCode>. <EuiCode>readModeProps</EuiCode> accepts any{' '}
```